test/suite_test.go

```diff
@@ -309,6 +325,29 @@ func (e *openShifte2e) execNodeOCP(node string, args ...string) string {
 	)
 }
 
+func (e *vanilla) SetupSuite() {
+	var err error
+	e.e2e.execNode = e.execNodeVanilla
+	e.kubectlPath, err = exec.LookPath("kubectl")
+	e.Nil(err)
+}
+
+func (e *vanilla) SetupTest() {
+	// Build and load the test image
+	e.logf("Building operator container image")
+	e.run("make", "image", "IMAGE="+e.testImage)
+	e.run(
+		containerRuntime, "pull", "quay.io/jaosorior/selinuxd",
+	)
+}
+
+func (e *vanilla) TearDownTest() {
+}
+
+func (e *vanilla) execNodeVanilla(node string, args ...string) string {
+	return e.run(containerRuntime, append([]string{"exec", node}, args...)...)
```

Comment:
Assuming we're already on the node (as `root`), we probably just have to call `bash`?

Reply:
Done